cmake/SSGCommon.cmake

```diff
@@ -271,7 +271,7 @@ macro(ssg_build_ansible_playbooks PRODUCT)
     set(ANSIBLE_PLAYBOOKS_DIR "${CMAKE_CURRENT_BINARY_DIR}/playbooks")
     add_custom_command(
         OUTPUT "${ANSIBLE_PLAYBOOKS_DIR}"
-        COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${SSG_BUILD_SCRIPTS}/build_rule_playbooks.py" --ssg-root "${CMAKE_SOURCE_DIR}" --product "${PRODUCT}"
+        COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${SSG_BUILD_SCRIPTS}/build_rule_playbooks.py" --ssg-root "${CMAKE_SOURCE_DIR}" --product "${PRODUCT}" --resolved-rules-dir "${CMAKE_CURRENT_BINARY_DIR}/rules" --output-dir "${CMAKE_CURRENT_BINARY_DIR}/playbooks"
```

Comment:
I would prefer to use `${ANSIBLE_PLAYBOOKS_DIR}` `"${CMAKE_CURRENT_BINARY_DIR}/playbooks"` because it is used as `OUTPUT` of this CMake command.

Reply:
that makes sense... I basically overlooked that this variable (`${ANSIBLE_PLAYBOOKS_DIR}`) was set

Reply:
fixed by rebase with force push